documentation/docs/scenarios/abapEnvironmentAddons.md

```diff
@@ -1,14 +1,24 @@
-# Build and Publish Add-on Products on SAP Cloud Platform ABAP Environment
+# Build and Publish Add-on Products on SAP BTP ABAP Environment
 
 ## Introduction
 
 !!! caution "Initial Scope"
     The initial scope supports an add-on product consisting of **one** software component. Furthermore, this software component can not be used in multiple add-on products.
     You cannot build add-ons based on gCTS branching functionality.
 
-This scenario describes how an add-on for the SAP Cloud Platform ABAP Environment is built. It is intended for SAP partners who want to provide a Software as a Service (SaaS) solution on the SAP Cloud Platform using the ABAP Environment. Therefore, a partner development contract (see [SAP PartnerEdge Test, Demo & Development Price List](https://partneredge.sap.com/en/library/assets/partnership/sales/order_license/pl_pl_part_price_list.html)) is required. This page aims to provide an overview of the build process of the add-on.
+!!! caution "Upcoming 2102 release of SAP BTP ABAP Environment"
+    With the upcoming 2102 release of SAP BTP ABAP Environment some changes to the backend behavior of the MANAGE_GIT_REPOSITORY service are introduced. Specifically:
+    - To pull a software component to a system, the software component needs to be cloned first.
+    - It is planned to add the possibility to clone a software component repeatedly with the hotfix collection HFC03 of release 2102
+    **Implications for the “abapEnvironmentPipeline”:**
+    If you are using the “Prepare System” stage to create a new ABAP Environment system, it is no longer possible to use the “Clone Repositories” stage with the “Pull” strategy or with the default strategy (no strategy specified). Please use the strategy “Clone” instead. For more information, have a look at the [stage documentation](../pipelines/abapEnvironment/stages/cloneRepositories.md).
+    The strategy “AddonBuild” will execute the abapEnvironmentCloneGitRepo instead of the previous logic. No configuration changes should be necessary.
+    Please be aware that a repeated execution of a pipeline using the strategy “Clone” or “AddonBuild” will not be possible until hotfix collection HFC03 (planned).
+    The recommended workaround is to replace the strategy “AddonBuild” with “CheckoutPull”, whenever the system from a previous pipeline run is reused.
```

Comment:
Just an idea for better readability: Maybe you can place these caution warnings between the headline 'Build and Publish Add-on Products on SAP BTP ABAP Environment' and 'Introduction'

Real introduction starts way more down